app/src/main/res/layout/activity_movie_details.xml

```diff
@@ -0,0 +1,318 @@
+        <ImageView
+            android:id="@+id/star_5_image_view"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="2dp"
+            android:layout_marginTop="8dp"
+            android:src="@drawable/ic_gray_star"
+            app:layout_constraintStart_toEndOf="@+id/star_4_image_view"
+            app:layout_constraintTop_toBottomOf="@+id/genere_text_view" />
+
+        <TextView
+            android:id="@+id/reviews_text_view"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="9dp"
+            android:text="@string/reviews_string"
+            android:textColor="@color/gray_color"
+            android:textSize="14sp"
+            android:textStyle="bold"
+            app:layout_constraintBottom_toBottomOf="@+id/star_5_image_view"
+            app:layout_constraintStart_toEndOf="@+id/star_5_image_view"
+            app:layout_constraintTop_toTopOf="@+id/star_5_image_view" />
+
+        <TextView
+            android:id="@+id/storyline_text_view"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="16dp"
+            android:layout_marginTop="24dp"
+            android:layout_marginEnd="16dp"
+            android:text="@string/storyline_string"
+            android:textColor="@color/storyline_text_color"
+            android:textSize="14sp"
+            android:textStyle="bold"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/reviews_text_view" />
+
+        <TextView
+            android:id="@+id/storyline_content_text_view"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="16dp"
+            android:layout_marginEnd="16dp"
+            android:alpha="0.75"
+            android:lineSpacingExtra="4sp"
+            android:text="@string/storyline_content_string"
+            android:textColor="@color/white"
+            android:textSize="14sp"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/storyline_text_view" />
+
+        <TextView
+            android:id="@+id/cast_text_view"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="16dp"
+            android:layout_marginTop="24dp"
+            android:text="@string/cast_string"
+            android:textColor="@color/storyline_text_color"
+            android:textStyle="bold"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/storyline_content_text_view" />
+
```

Comment:
Try to make the view with rounded corners, not use image assets with rounded corners.